obsplus/utils.py

```diff
@@ -839,6 +846,64 @@ def _get_distance_tuple(obj):
     return df[list(DISTANCE_COLUMNS)]
 
 
+def calculate_distance(
```

Comment:
This is a good addition. It should probably replace/augment the inner functions defined in `get_distance_df` but that doesn't have to be done in this PR.

Reply:
I might leave that up to you - I wasn't sure what `get_distance_df` was going to return.